### DISCUSSION TITLE
It's especially slow when you close a non-last tab

### DISCUSSION THREAD

**Discussion by naosense** [wezterm/wezterm#5298] [General]
For example, if you have three tabs, tab1, tab2, and tab3, the speed of closing tab3 is normal, but when you close the first two tabs, the speed is very slow, about 10s

**Comment by nekowinston** [comment]
This should probably be an issue with additional setup info and logs attached.

**Reply by naosense** [reply]
> This should probably be an issue with additional setup info and logs attached.

Thanks for your reply! What's that mean?

**Reply by nekowinston** [reply]
See the [bug issue form](https://github.com/wez/wezterm/issues/new?assignees=&labels=bug&projects=&template=bug.yml)

**Reply by naosense** [reply]
i submit an issue here #5304
